### DISCUSSION TITLE
Filament Sensor does not trigger filament change.

### DISCUSSION THREAD

**Discussion by VashBaldeus** [Jyers/Marlin#518] [Q&A] [answered]
Filament Sensor does not trigger filament change. I have a filament sensor that after install & testing worked with the firmware and did trigger alert to change, but for long prints it does not trigger alert to change filament when time comes and filament is long past the sensor.

**Comment by Jyers** [comment, marked as answer]
This is an issue for the main marlin repo, not this fork. I haven't changes anything about how filament runout is handled.
